src/amo/components/AddonDetail.js

```diff
@@ -161,6 +161,10 @@ export class AddonDetailBase extends React.Component {
     const authorList = addon.authors.map(
       (author) => `<a href="${author.url}">${author.name}</a>`);
 
+    const descriptionSanitized = addon.description && addon.description.length ?
```

Comment:
Isn't `addon.description.length` redundant? An empty string will always be falsey.